release_notes/RELEASE.txt

```diff
@@ -53,8 +53,18 @@ Support for new platforms and compilers
 =======================================
 
 
-Bugs fixed since HDF 4.2.15
+Bugs fixed since HDF 4.2.16
```

Comment:
This should probably be since 4.2.16-1

Reply:
But isn't this patch 4.2.16-1?

Reply:
Maybe this is 4.2.16-2, but you're right, the previous release was version 4.2.16.

1.